wovnrb.gemspec

```diff
@@ -12,6 +12,7 @@ Gem::Specification.new do |spec|
   spec.homepage      = 'https://wovn.io'
   spec.license       = 'MIT'
   spec.required_ruby_version = '>= 2.5'
+  spec.metadata['rubygems_mfa_required'] = 'true'
```

Comment:
RuboCop wants this. More info:

- https://docs.rubocop.org/rubocop/cops_gemspec.html#gemspecrequiremfa
- https://guides.rubygems.org/mfa-requirement-opt-in/

Is this OK?